lms/djangoapps/verify_student/tasks.py

```diff
@@ -0,0 +1,29 @@
+"""
+Django Celery tasks for service status app
+"""
+import logging
+from smtplib import SMTPException
+
+from celery import task
+from django.conf import settings
+from django.core.mail import send_mail
+
+ACE_ROUTING_KEY = getattr(settings, 'ACE_ROUTING_KEY', None)
+log = logging.getLogger(__name__)
+
+
+@task(routing_key=ACE_ROUTING_KEY)
+def send_verification_status_email(subject, message, from_addr, dest_addr):
+    """
+    Spins a task to send verification status email to the learner
+    """
+    try:
+        send_mail(
```

Comment:
Why are we not using Sailthru?

Reply:
It seems like all this task is ding is to send an email and there is nothing specific to verification. So may be you can change the name to something more generic and not include verification_status in the name, This is just a suggestion if you want to keep it this way it is fine as well.

Reply:
According to the new requirement mentioned in LEARNER-5931 there should be a separate template for the verification status email in edx-platform that's why there is no need of sail-thru.
For the modification of function name; Yeah, I will follow your suggestion!